src/lightning_fabric/CHANGELOG.md

```diff
@@ -24,6 +23,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   * Added `Fabric.loggers` and `Fabric.logger` attributes to access the individual logger instances
   * Added support for calling `self.log` and `self.log_dict` in a LightningModule when using Fabric
   * Added access to `self.logger` and `self.loggers` in a LightningModule when using Fabric
+  * Added `lightning_fabric.loggers.TensorBoardLogger`
+- Added `lightning_fabric.loggers.CSVLogger` ([#16346](https://github.com/Lightning-AI/lightning/issues/16346))
```

Comment:
Should this be indented and with `*`?

Reply:
Probably not, because the above group of items was all done in one PR.